src/pylorax/api/queue.py

```diff
@@ -676,10 +678,24 @@ def uuid_log(cfg, uuid, size=1024):
     if status is None:
         raise RuntimeError("Status is missing for %s" % uuid)
 
-    if status["queue_status"] == "RUNNING":
-        log_path = "/tmp/anaconda.log"
-    else:
-        log_path = joinpaths(uuid_dir, "logs", "combined.log")
+    def get_log_path():
+        # Try to return the most relevant log at any given time during the
+        # compose. If the compose is not running, return the composer log.
+        if status["queue_status"] != "RUNNING":
+            return joinpaths(uuid_dir, "logs", "combined.log")
+        anaconda_log = "/tmp/anaconda.log"
+        packaging_log = "/tmp/packaging.log"
+        if not (os.path.isfile(anaconda_log) and os.path.isfile(packaging_log)):
+            return anaconda_log
+        anaconda_mtime = os.stat(anaconda_log).st_mtime
+        packaging_mtime = os.stat(packaging_log).st_mtime
+        # If the packaging log exists and its last message is at least 15
+        # seconds newer than the anaconda log, return the packaging log.
+        if packaging_mtime > anaconda_mtime + 15:
+            return packaging_log
+        return anaconda_log
+    log_path = get_log_path()
+
```

Comment:
Also, this block should be wrapped in a try/except that returns the combined log -- the anaconda logs could vanish at any time.